### DISCUSSION TITLE
Recipe's CSS is not generated at build time when setting the include option to specific path.

### DISCUSSION THREAD

**Discussion by erm1116** [chakra-ui/panda#1224] [Q&A] [answered]
I want to use Panda CSS recipes only in the specific directory.
So, I tried to set the `include` option like `./src/ui/components/**/*.{js,jsx,ts,tsx}`, but the css used in the pre-defined recipe is not generated at build time.
When setting the option like `./src/**/*.{js,jsx,ts,tsx}` works well.

Is there a good way to do it?

---
```ts
// panda.config.ts
export default defineConfig({
  preflight: true,
  include: ["src/ui/components/**/*.{js,jsx,ts,tsx}"],
  exclude: [],
  theme: {
    extend: {
      recipes: {
        checkbox: checkboxRecipe,
      },
    },
  },
  outdir: "./src/ui/styled-system",
})
```

**Comment by astahmer** [comment, marked as answer]
for future folks stumbling on that issue: this is expected, recipe are JIT generated

> Technical Notes 📝: Only the recipe and variants used in your application are generated. Not more!

https://panda-css.com/docs/concepts/recipes#using-the-recipe-1

you need to either:
- use `cva` that will always be generated, produces atomic classNames
- or use `config.staticCss` https://panda-css.com/docs/guides/static#generating-recipes
